available_images.md

```diff
@@ -0,0 +1,135 @@
+|  			 Asia Pacific (Singapore) 			  |  			 ap-southeast-1 			 |  			 Available 			         |  			 None 			                        |  			 763104351884.dkr.ecr.ap-southeast-1.amazonaws.com/&lt;repository-name>:&lt;image-tag> 			   |
+|  			 Asia Pacific (Sydney) 			     |  			 ap-southeast-2 			 |  			 Available 			         |  			 None 			                        |  			 763104351884.dkr.ecr.ap-southeast-2.amazonaws.com/&lt;repository-name>:&lt;image-tag> 			   |
+|  			 Middle East (Bahrain) 			     |  			 me-south-1 			     |  			 Available 			         |  			 None 			                        |  			 217643126080.dkr.ecr.me-south-1.amazonaws.com/&lt;repository-name>:&lt;image-tag> 			       |
+|  			 South America (Sao Paulo) 			 |  			 sa-east-1 			      |  			 Available 			         |  			 None 			                        |  			 763104351884.dkr.ecr.sa-east-1.amazonaws.com/&lt;repository-name>:&lt;image-tag> 			        |
+|  			 Canada (Central) 			          |  			 ca-central-1 			   |  			 Available 			         |  			 None 			                        |  			 763104351884.dkr.ecr.ca-central-1.amazonaws.com/&lt;repository-name>:&lt;image-tag> 			     |
+|  			 EU (Frankfurt) 			            |  			 eu-central-1 			   |  			 Available 			         |  			 None 			                        |  			 763104351884.dkr.ecr.eu-central-1.amazonaws.com/&lt;repository-name>:&lt;image-tag> 			     |
+|  			 EU (Stockholm) 			            |  			 eu-north-1 			     |  			 Available 			         |  			 None 			                        |  			 763104351884.dkr.ecr.eu-north-1.amazonaws.com/&lt;repository-name>:&lt;image-tag> 			       |
+|  			 EU (Ireland) 			              |  			 eu-west-1 			      |  			 Available 			         |  			 Available 			                   |  			 763104351884.dkr.ecr.eu-west-1.amazonaws.com/&lt;repository-name>:&lt;image-tag> 			        |
+|  			 EU (London) 			               |  			 eu-west-2 			      |  			 Available 			         |  			 None 			                        |  			 763104351884.dkr.ecr.eu-west-2.amazonaws.com/&lt;repository-name>:&lt;image-tag> 			        |
+|  			 EU (Paris) 			                |  			 eu-west-3 			      |  			 Available 			         |  			 None 			                        |  			 763104351884.dkr.ecr.eu-west-3.amazonaws.com/&lt;repository-name>:&lt;image-tag> 			        |
+|  			 China (Beijing) 			           |  			 cn-north-1 			     |  			 Available 			         |  			 None 			                        |  			 727897471807.dkr.ecr.cn-north-1.amazonaws.com.cn/&lt;repository-name>:&lt;image-tag> 			    |
+|  			 China (Ningxia) 			           |  			 cn-northwest-1 			 |  			 Available 			         |  			 None 			                        |  			 727897471807.dkr.ecr.cn-northwest-1.amazonaws.com.cn/&lt;repository-name>:&lt;image-tag> |
+
+
+Note: **eu-north-1** is available starting in the version 2.0 release.
```

Comment:
Now that the versioning scheme is changing, can we change this wording to emphasize the framework versions available in this region?

Reply:
Sure, do you have a link to information on the new versioning scheme?

Reply:
Added column with the latest version